From 8f54e59b75dec2503805702e716dc01768914af8 Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Thu, 5 Feb 2015 12:26:56 +0800
Subject: [PATCH] MLK-10203-3 arm: pcie: add the disp mix pwr domain

since the pcie is power-ed by disp_mix domain,
add disp_mix domain into pcie device node.

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index a72fb36..51d91c9 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1487,6 +1487,7 @@
 				 <&clks IMX6SX_CLK_DISPLAY_AXI>;
 			clock-names = "pcie", "pcie_bus", "pcie_phy", "pcie_inbound_axi";
 			pcie-phy-supply = <&reg_pcie_phy>;
+			power-domains = <&gpc 2>;
 			status = "disabled";
 		};
 	};
-- 
1.8.0

